### DISCUSSION TITLE
How to expose next_auth API schema on a self-hosted docker setup ?

### DISCUSSION THREAD

**Discussion by mikekuzak** [supabase/supabase#19141] [General]
The documentation says:

"Expose the NextAuth schema in Supabase
Expose the next_auth schema via the Serverless API in the [API settings](https://app.supabase.com/project/_/settings/api) by adding next_auth to the "Exposed schemas" list."

There is no API Settings section in the self-hosted version. 
Also there is a remark about supabase-cli but I don't know how this fits into the docker-compose setup. 
I tried to find the the supabase/config.yml in any of the containers. No luck.

**Comment by znowfox** [comment]
so its postgrest, you can add your schema name to the schema list in the .env file (if running through docker anyways)  ex: PGRST_DB_SCHEMAS=public,storage,graphql_public,mynewschema

rebuild.
